PWGCF/FemtoUniverse/Tasks/femtoUniversePairTaskTrackV0Extended.cxx

```diff
@@ -268,6 +270,21 @@ struct femtoUniversePairTaskTrackV0Extended {
       pairCloseRejection.init(&resultRegistry, &qaRegistry, ConfCPRdeltaPhiCutMin.value, ConfCPRdeltaPhiCutMax.value, ConfCPRdeltaEtaCutMin.value, ConfCPRdeltaEtaCutMax.value, ConfCPRChosenRadii.value, ConfCPRPlotPerRadii.value);
       pairCloseRejectionV0.init(&resultRegistry, &qaRegistry, ConfCPRdeltaPhiCutMin.value, ConfCPRdeltaPhiCutMax.value, ConfCPRdeltaEtaCutMin.value, ConfCPRdeltaEtaCutMax.value, ConfCPRChosenRadii.value, ConfCPRPlotPerRadii.value);
     }
+
+    if (!ConfLocalEfficiency.value.empty()) {
+      TFile* pf = TFile::Open(ConfLocalEfficiency.value.c_str(), "read");
+      if (!pf || pf->IsZombie())
+        LOGF(fatal, "Could not load efficiency histogram from %s", ConfLocalEfficiency.value.c_str());
+      if (doprocessSameEvent || doprocessMixedEvent) {
+        plocalEffp1 = (ConfChargePart1 > 0) ? std::unique_ptr<TH1>(static_cast<TH1*>(pf->Get("PrPlus"))) : std::unique_ptr<TH1>(static_cast<TH1*>(pf->Get("PrMinus"))); // note: works only for protons for now
```

Comment:
Also, I wonder if with ROOT we can avoid static_casts to raw pointers... I am not sure.
I can conditionally accept this part for now.

Reply:
I came across a template method Get() in the ROOT directory. This should do for now. Thanks.